src/offline/cable_driver_common.F90

```diff
@@ -328,4 +331,127 @@ SUBROUTINE cable_driver_init_cru(dels, koffset, CRU)
 
   END SUBROUTINE cable_driver_init_cru
 
-END MODULE cable_driver_init_mod
+  SUBROUTINE prepareFiles(ncciy)
+    USE cable_IO_vars_module, ONLY: logn,gswpfile
```

Comment:
This is going in a new file so I'd like to have minimal, FORD-compatible documentation.

```suggestion
  SUBROUTINE prepareFiles(ncciy)
    !* Select the correct files given the year for filenames following the gswp format
    
    USE cable_IO_vars_module, ONLY: logn,gswpfile
```